### PR TITLE
Added galician openoffice

### PR DESCRIPTION
**Pull request by alexandregz**
sorry for delay, I make new updated branch

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Same problems [as the portuguese one](https://github.com/caskroom/homebrew-versions/pull/819).

**Comment by alexandregz** [comment]
Done

**Comment by vitorgalvao** [comment]
Thank you, but still need to [squash the commits](http://davidwalsh.name/squash-commits-git).

**Comment by alexandregz** [comment]
rebased

**Comment by alexandregz** [comment]
squashed

File: Casks/openoffice-gl.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask :v1 => 'openoffice-gl' do
+  version '4.1.1'
+  sha256 'a41bcfbd6aeaf86c65ec27db78ad0fa8e93482dbc75336557fed7a42c8234d37'
+
+  # sourceforge.net is the official download host per the vendor homepage
+  url "http://downloads.sourceforge.net/project/openofficeorg.mirror/#{version}/binaries/gl/Apache_OpenOffice_#{version}_MacOS_x86-64_install_gl.dmg"
+  name 'OpenOffice
```
Comment (vitorgalvao) [inline Casks/openoffice-gl.rb]: You forgot the closing quote.

**Comment by alexandregz** [comment]
oooops! done, thx!

**Pull request merged**